torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -78,6 +78,20 @@ def _get_default_op_backend_config(op, dtype_configs):
         torch.nn.functional.layer_norm,
     ]]
 
+_ADD_CONFIG = {
+    "pattern": operator.add,
+    "num_tensor_args_to_observation_type": {
+        # TODO: maybe change this to NO_OBSERVER
+        0: ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
```

Comment:
nit: does the zero case actually appear, or does FX inline the calculation?

Reply:
if there are zero tensors, should we not insert observers then? Observers expect a tensor, violating this will lead to a runtime error.

Reply:
oh, yeah we do not insert observer actually, this is not really used right now since we have some extra checks in prepare: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/prepare.py#L316, we can change this to NO_OBSERVER later after we have dtype inference implemented, maybe I can add more comments for TODO